charts/jd-public/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+name: jd-public
+apiVersion: v2
+version: 0.1.0
+appVersion: "2.15.129"
+description: Juror Digital - Public
+maintainers:
+  - name: HMCTS Platform Operations
```

Comment:
I would remove this we don't maintain it

Reply:
removed the entry